### ISSUE TITLE
Wireguard doesn't work

### ISSUE THREAD

**Issue by c25** [cb-linux/breath#315]
This is *probably* due to kernel headers being missing/mismatched.

* Install wireguard via apt (this is tested on debian unstable)
* configure your connection
* run `wg-quick up wg0`
* receive the following error(s)

```
Error: Unknown device type.
Unable to access interface: Protocol not supported
```

This appears to be because we're lacking the wireguard kernel module.

installing `wireguard-dkms` (or running `dpkg-reconfigure wireguard-dkms`) fails/skips as the kernel headers for the running kernel are not installed.

```bash
$uname -r
5.10.106
```

`linux-headers-amd64 5.18.16-1` are installed

Even if we could install the correct version I doubt they would match the running (chromebook) kernel.

**Comment by jkaplavka** [comment]
Yes it's custom kernel modified by google so mainline headers doesn't work, but since kernel 5.6 there is a wireguard module build in. I'm on kernel 5.15 with working wireguard. Try this `sudo modprobe wireguard` if that doesn't work you will have to change kernel config and rebuild kernel with wireguard module https://cb-linux.github.io/breath/docs.html#/kernel?id=new-build-script, this config needs to be enabled https://www.kernelconfig.io/search?q=CONFIG_WIREGUARD&kernelversion=5.10.137&arch=x86
